City-of-Helsinki/drupal-infofinland
# UHF-10279

# [UHF-10279](https://helsinkisolutionoffice.atlassian.net/browse/UHF-10279)
Solved a problem with jsonapi 

## What was done
Fixed the problem


## How to install

* Make sure your instance is up and running on correct branch.
  * `git checkout UHF-10279`
  * `make fresh`
* Run `make drush-cr`

## How to test
- Setup the frontend as stated in [infofinland-ui readme](https://github.com/City-of-Helsinki/infofinland-ui/blob/dev/README.md)
- During the build, there should be no errors
- Start browsing the site with your network-tab open, keep an eye out for errors
  - Also check that there are no errors in the terminal where you ran yarn dev
- Create new page, open it and there should be no errors.
